src/bindings/capi/thorvg_capi.h

```diff
@@ -2146,6 +2146,7 @@ TVG_API Tvg_Saver* tvg_saver_new();
 * \param[in] paint The paint to be saved with all its associated properties.
 * \param[in] path A path to the file, in which the paint data is to be saved.
 * \param[in] compress If @c true then compress data if possible.
```

Comment:
is compress still relevant ?

check doxygen output for similar inconstancy

Reply:
Thanks!